packages/express/mixins/mixin.core.js

```diff
@@ -12,6 +15,10 @@ const {
 } = require('@untool/core');
 
 class ExpressMixin extends Mixin {
+  constructor(...args) {
+    super(...args);
+    this.enhancedPromise = new EnhancedPromise();
```

Comment:
Could we name this instance variable `serverAddress` or similar instead of `enhancedPromise`, so that we actually know what is in it?

Reply:
We could call it `serverAddressPromise`, but I decided to keep the naming in line with what we already have: https://github.com/untool/untool/blob/5054511e37a7cb3178f950396d1fde1b8656698a/packages/webpack/mixins/stats/mixin.core.js#L18-L25